cpu/cpu_by_core.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+usa=$(grep -w 'cpu' /proc/stat | awk '{usage=($2+$3+$4)*100/($2+$3+$4+$5+$6+$7+$8)} END {print usage"%"}')
+
+echo "$usa"
```

Comment:
is this file required?

Reply:
The functionality of this file has been updated and even renamed to